### PR TITLE
HOLO-1022: Clean up statically defined and unused variables

### PR DESCRIPTION
**Pull request by alexanderattar**
## Describe Changes

- Cleanup a lot of unused statically defined variables (we'll eventually be able to import anything we need like this from the SDK as we need it)
- Move web3 specific functions into the web3 file instead of generic utils file
- Use getABIs instead of __dir
- Add more type safety
- Got rid of all uses of `await fs.readJson(path.join(__dirname, `../abi/${environment}/<ContractName>.json`)` except for in the reusable utility function `getABIs`

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] I have checked eslint

### CODE REVIEW

File: src/utils/web3.ts [hunk 1]
```diff
@@ -1,133 +1,27 @@
...
-  : SUPPORTED_TESTNET_CHAIN_IDS
-export const SUPPORTED_CHAIN_NAMES = SUPPORTED_CHAIN_IDS.map(id => CHAIN_NAMES[id])
-export const SUPPORTED_NETWORK_NAMES = SUPPORTED_CHAIN_IDS.map(id => CHAIN_NETWORKS[id])
-export const DISABLED_CHAIN_IDS = []
-export const DISABLED_CHAIN_NAMES = DISABLED_CHAIN_IDS.map(id => CHAIN_NAMES[id])
-
-export const DROPDOWN_CHAIN_IDS = [...SUPPORTED_CHAIN_IDS, C.disconnect_action]
-export const DROPDOWN_CHAIN_NAMES = DROPDOWN_CHAIN_IDS.map(id => CHAIN_NAMES[id])
+
+export const networkToChainId: Readonly<Record<string, number>> = Object.fromEntries(
```
Comment (sogoiii) [inline src/utils/web3.ts:12]: This is a future thing, but i feel like `networkToChainId`, `chainIdtoNetwork` and `NETWORK_COLORS` should be in the networks package.

File: src/utils/network-monitor.ts [hunk 2]
```diff
@@ -707,67 +729,46 @@ export class NetworkMonitor {
       this.activated[network] = true
     }
 
-    const holographABI = await fs.readJson(path.join(__dirname, `../abi/${this.environment}/Holograph.json`))
+    // Setup contracts
+    const abis = await getABIs(this.environment)
```
Comment (sogoiii) [inline src/utils/network-monitor.ts:714]: @alexanderattar Glad this direct `readJson` fn is finally abstracted away.

File: src/utils/network-monitor.ts [hunk 3]
```diff
@@ -707,67 +729,46 @@ export class NetworkMonitor {
...
+      cxipERC721Contract: {address: this.cxipERC721Address, abi: abis.CxipERC721ABI},
+      bridgeContract: {address: this.bridgeAddress, abi: abis.HolographBridgeABI},
+      factoryContract: {address: this.factoryAddress, abi: abis.HolographFactoryABI},
+      interfacesContract: {address: this.interfacesAddress, abi: abis.HolographInterfacesABI},
+      operatorContract: {address: this.operatorAddress, abi: abis.HolographOperatorABI},
+      registryContract: {address: this.registryAddress, abi: abis.HolographRegistryABI},
+      messagingModuleContract: {address: this.messagingModuleAddress, abi: abis.LayerZeroABI},
+    }
+
+    for (const contractName of Object.keys(contractsMap)) {
```
Comment (sogoiii) [inline src/utils/network-monitor.ts:746]: @alexanderattar i love this! Makes the original code shorter.

File: src/utils/contracts.ts [hunk 4]
```diff
@@ -18,9 +37,11 @@ export const getABIs = async (environment: string): Promise<any> => {
     HolographOperatorABI: await fs.readJson(path.join(__dirname, `../abi/${environment}/HolographOperator.json`)),
     HolographRegistryABI: await fs.readJson(path.join(__dirname, `../abi/${environment}/HolographRegistry.json`)),
     LayerZeroABI: await fs.readJson(path.join(__dirname, `../abi/${environment}/LayerZeroEndpointInterface.json`)),
+    MockLZEndpointABI: await fs.readJson(path.join(__dirname, `../abi/${environment}/MockLZEndpoint.json`)),
```
Comment (sogoiii) [inline src/utils/contracts.ts:60]: glad this and OwnerABI is now in this object.

File: src/utils/web3.ts (hunk 1 above)
Comment (sogoiii) [reply src/utils/web3.ts:12]: @alexanderattar Oh and when we do move it, I think we should throw an error if someone tries to access a key that does not exist.

**Review by sogoiii — APPROVED** [review]
👍

File: src/utils/web3.ts (hunk 1 above)
Comment (alexanderattar) [reply src/utils/web3.ts:12]: yeah that's a good idea

**Pull request merged**